geometric/optimize.py

```diff
@@ -118,6 +122,13 @@ def __init__(self, coords, molecule, IC, engine, dirname, params, print_info=Tru
         self.lowq_tr_limit = 1
         # Recalculate the Hessian after a trigger - for example if the energy changes by a lot during TS optimization.
         self.recalcHess = False
+        # IRC related attributes
+        self.IRC_direction = 1
```

Comment:
Could these attributes be combined into a dictionary such as `self.IRC_info = {'IRC_direction': 1 ... }`?

Reply:
[Fixed](Yes, since it will vary the stepsize based on the step quality, I set the `tmax` equal to `trust` so stepsize won't exceed the `trust`.)

Reply:
[Fixed](https://github.com/leeping/geomeTRIC/pull/197/files#diff-5da1085de6679e8eb0ed65e4cf06827a071fa48dff306e1645ce2f19779728bbR126)

Reply:
The attributes are combined into a dictionary.